DYDX-Bots/Insolvent_Contract/src/findings.ts

```diff
@@ -0,0 +1,17 @@
+import { Finding, FindingType, FindingSeverity } from "forta-agent";
+import { BigNumber } from "ethers";
+
+export const createFinding = (totalBorrowerDebtBalance: BigNumber, totalActiveBalanceCurrentEpoch: BigNumber) => {
+  return Finding.fromObject({
+    name: "Contract Insolvent",
+    description: "Total borrower debt balance has exceeded total active balance current epoch",
```

Comment:
```suggestion
    description: "The total borrowed balance has exceeded total active balance in the current epoch",
```
The current one is fine, this is a suggestion to make the description more inspired from the docs rather than the variables/events names. Using "Total borrowed balance" seems more understandable than Total borrower debt balance.